app/src/main/java/com/keylesspalace/tusky/components/announcements/AnnouncementsViewModel.kt

```diff
@@ -56,8 +57,9 @@ class AnnouncementsViewModel @Inject constructor(
             appDatabase.instanceDao().loadMetadataForInstance(accountManager.activeAccount?.domain!!)
                 .map<Either<InstanceEntity, Instance>> { Either.Left(it) }
                 .onErrorResumeNext {
-                    mastodonApi.getInstance()
-                        .map { Either.Right(it) }
+                    rxSingle {
```

Comment:
honestly should just replace `mastodonApi.getCustomEmojis(),` with coroutines too and just remove this whole mess

Reply:
I was planning to do this in another PR, but I can also do it with this one

Reply:
I think either way is fine